roles/mockturtle/files/sopel_modules/invite.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+# vi: set fileencoding=utf-8
+
+import time
+import sopel.bot as bot
+import sopel
+
+@sopel.module.rule(r'.*[Ii]nvite.*')
+@sopel.module.rate(900)
+def invite(bot, trigger):
+    """To new members that are looking for an invitation"""
+    joinedTime = bot.db.get_nick_value(trigger.nick, 'join_channel')
+    diff = time.gmtime() - joinedTime
+    if diff < 3600:
+        bot.reply(
+            "New contributors are always welcome on Lobsters. If you are the
+author or otherwise involved with a story that was submitted to the
+site, ask and someone will invite you. If not, offer some good links
+or thoughts on a story and someone will likely invite you so can post
+them.  Alternatively, share a link to your homepage or public profile
+(GitHub, Twitter, YC News and LinkedIn are popular choices) so people
+can see you're not a spammer. https://lobste.rs/chat")
```

Comment:
This text turns out to be slightly too long for IRC.  I know I gave you guidance to use it--will you remove the end:

"so people can see you're not a spammer. https://lobste.rs/chat"

and just end with a period.

After doing so, will you add a pair of double quotes to each line?  Right now the carriage return is embedded in the string.  We don't want that.  Be careful to leave a space at the end of each line so the word ending one line doesn't join the word starting the next line.

"first line "
"second line "
"all but the last ending with a space."

Reply:
After discussion on IRC, let's use the invite text as we had it previously:

If you would like an invite to lobste.rs, please look at the chat FAQ first. Thank you. https://lobste.rs/chat (I am a bot. (v).v.(v))